ui/src/views/AutogenView.vue

```diff
@@ -679,7 +679,6 @@ export default {
     },
     getOkProps () {
       if (this.selectedRowKeys.length > 0 && this.currentAction?.groupAction) {
-        return { props: { type: 'default' } }
```

Comment:
Why do we need to remove this?

Reply:
it is now in the edit dialog (we don't need to)